infra/charts/turing/templates/cluster-role-binding.yaml

```diff
@@ -3,6 +3,14 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: {{ template "turing.serviceAccount.name" . }}-role-binding
+  labels:
+    app: {{ template "turing.fullname" . }}
+    chart: {{ template "turing.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
It could make sense to standardize this under `_helpers.tpl` and then include it in the individual k8s resources.
Something like: 
```
labels:
  {{ toYaml (template "turing.labels" .) | indent 4 -}}
```